connectors/sources/__init__.py

```diff
@@ -1,5 +0,0 @@
-#

```

Comment:
This file does not seem needed and it triggers weird linter error, so I decided to delete